src/scripts/generate-readme.ts

```diff
@@ -0,0 +1,106 @@
+import fs from 'node:fs'
```

Comment:
Node has a nice fs promised based API now :)
```suggestion
import { readFile, writeFile } from 'node:fs/promises'
```

Reply:
Yep, but we're not using any callbacks. Is there a difference with promises vs. sync methods?

Reply:
Just coming from async first. Sync IMO was always meant as an escape hatch when forced to work in a sync context. For example, they didn't name the methods readFile and readFileAsync

Reply:
fair enough. Thanks for the explanation, will update this 👍